src/calculateRentalCost.js

```diff
@@ -4,7 +4,12 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
```

Comment:
Use `const` for the daily cost value instead of hardcoding it to improve code maintainability.

Reply:
not fixed